src/configs/dbConfig.ts

```diff
@@ -0,0 +1,25 @@
+import { DataSource } from 'typeorm';
+
+export const dataSource = new DataSource({
+  type: 'postgres',
+  host: process.env.DB_HOST,
+  port: parseInt(process.env.DB_PORT || '3306'),
+  username: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_NAME,
+  entities: ['src/entity/*.js'],
+  logging: true,
+  synchronize: true,
+});
+
+export async function testConnection() {
```

Comment:
Shall we rename the method to something like `initConnection` and reuse that function in both `app.ts` and `db.test.ts`?